packages/sdk/src/build/build-get-account.js

```diff
@@ -1,11 +1,12 @@
 import {pipe, makeGetAccount, Ok} from "../interaction/interaction.js"
 import {sansPrefix} from "@onflow/util-address"
 
-export function getAccount(addr) {
+export function getAccount(addr, atSealedBlock = false) {
```

Comment:
for a frequency point of view, most users of the chain would actually want the account at the sealed block, so i wonder if that should actually be the default here

Reply:
I confess I am little confused here, what is the use case of `false` here if it can maybe fail ?